cloudevents/working-drafts/dds-protocol-binding.md

```diff
@@ -0,0 +1,266 @@
+elements defined in the CloudEvents specification are implemented in the DDS protocol
+and transmitted as [DDS messages][dds-message-format].
+
+### 1.1. Conformance
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC2119][rfc2119].
+
+### 1.2. Relation to DDS
+
+This specification does not prescribe rules constraining transfer or settlement
+of event messages with DDS; it solely defines how CloudEvents are expressed in
+the DDS protocol as [DDS messages][dds-message-format].
+
+The DDS protocol is an [Object Management Group (OMG)][omg] middleware standard
+for efficient and real-time data exchange in distributed systems.
+It enables communication using a publish-subscribe model and
+operates in conjunction with the [Real-Time Publish Subscribe (RTPS)][rtps]
+wire protocol.
+DDS ensures reliable data delivery for latency-sensitive payloads,
+supporting Quality of Service (QoS) parameters like reliability, lifespan
+and partitioning. DDS is most commonly used for mission-critical applications
+in industries such as aerospace, healthcare, and industrial automation.
+
+This binding specification defines how attributes and data of a CloudEvent is
+mapped to a DDS message format.
+
+### 1.3. Content Modes
+
+The CloudEvents specification defines three content modes for transferring
+events: _structured_, _binary_ and _batch_. 
+
+Every compliant implementation of the CloudEvent specification SHOULD support
+both structured and binary modes. The DDS protocol binding supports both these modes.
+
+The DDS protocol binding does not currently support the _batch_ content mode.
+
+In the _structured_ content mode, event metadata attributes and event data are
+placed into the DDS message using an [event format](#14-event-formats).
+
+In the _binary_ content mode, the value of the event `data` is placed into
+the DDS message's `data` section as-is; all other event attributes are
+mapped to the DDS message's event metadata fields.
+
+### 1.4. Event Formats
+
+Event formats define how an event is expressed in a particular data format.
+All implementations of this specification that support the _structured_
+content mode MUST support the [JSON event format][json-format].
+
+The DDS protocol binding additionally supports the
+[DDS event format][dds-message-format].
+
+### 1.5. Security
+
+This specification does not introduce any new security features for DDS, or
+mandate specific existing features to be used.
+
+## 2. Use of CloudEvents Attributes
+
+This specification does not further define any of the [CloudEvents][ce] event
+attributes.
+
+### 2.1. data
+
+`data` is assumed to contain opaque application data that is encoded as declared
+by the `datacontenttype` attribute. 
+
+There are two values of `datacontenttype` currently supported:
+- `cloudevent/json` for JSON data
```

Comment:
Unless I'm mis-understanding this statement it doesn't make sense to me ... iis the suggestion that I can have a receive a could-event over-the-wire that has a data payload which represents another cloud-event represented in JSON ?

I'm not clear why this needs to be called out explicitly as clouds-events can wrap *any* content .. what's special about json in this context.